spiderList/spiderList.xml

```diff
@@ -703,6 +715,10 @@
 	<spider ident="legs">
 		<name>legs</name>
 	</spider>
+	<spider ident="Let's Encrypt validation server">
```

Comment:
I don't get, why this is should be included in this list? This is not a bot, which crawls your site. 
Actually, the bot should not come into contact with your page either.